### DISCUSSION TITLE
/v2/zh/guide/blog/article.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#198] [Announcements]
# /v2/zh/guide/blog/article.html

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/blog/article.html

**Comment by DrAugus** [comment]
能否全局设置所有文章为非博文。我发现我的所有页面都出现在文章列表里。

**Reply by Mister-Hope** [reply]
https://vuepress-theme-hope.github.io/v2/zh/config/plugins/blog.html#filter
